spec.yaml

```diff
@@ -759,3 +762,59 @@ paths:
           description: Error
           schema:
             $ref: "definitions.yaml#/definitions/V4GenericResponse"
+
+  /v4/releases/:
```

Comment:
The name is fine for me. No strong feelings. We should only make sure at some point to align the wording with our internal code in case this will be established.